docs/application/toc_all.md

```diff
@@ -89,6 +89,16 @@
 ##### [Overview](/application/dotnet/guides/nui/animation.md)
 ##### [Animation Types](/application/dotnet/guides/nui/animation-types.md)
 #### [Resources](/application/dotnet/guides/nui/resources.md)
+### NUI Components
```

Comment:
I'm confused with this depth.

```
### Natural User Interface
#### [Overview](/application/dotnet/guides/nui/overview.md)
...
#### [Layer](/application/dotnet/guides/nui/layer.md)
#### UI Components
##### [Overview](/application/dotnet/guides/nui/ui-components.md)
...
##### [VideoView](/application/dotnet/guides/nui/videoview.md)
#### Animation
##### [Overview](/application/dotnet/guides/nui/animation.md)
...
#### [Resources](/application/dotnet/guides/nui/resources.md)
### NUI Components
#### [Button](/application/dotnet/guides/nui/nui-components/Button.md)
#### [Loading](/application/dotnet/guides/nui/nui-components/Loading.md)
#### [Popup](/application/dotnet/guides/nui/nui-components/Popup.md)
#### [Progress](/application/dotnet/guides/nui/nui-components/Progress.md)
#### [Scrollbar](/application/dotnet/guides/nui/nui-components/Scrollbar.md)
#### [Slider](/application/dotnet/guides/nui/nui-components/Slider.md)
#### [Switch](/application/dotnet/guides/nui/nui-components/Switch.md)
#### [Tab](/application/dotnet/guides/nui/nui-components/Tab.md)
#### [Toast](/application/dotnet/guides/nui/nui-components/Toast.md)
```

According to this depth, the "NUI Components" should be in same depth with "Natural User Interface".
![image](https://user-images.githubusercontent.com/9958374/67282759-68044100-f50d-11e9-8438-5de144b22eaa.png)

Would you please confirm the depth of this NUI components?

Reply:
"NUI Components" should be below "Natural User Interface". And it replace "UI Components". 
@Seoyeon2Kim please confirm name of the title.

Reply:
'NUI Components' will improve the current 'UI Components'.
So, all categories of 'NUI Components' should be under 'Natural User Interface'.

@junqingma 
1. Please move all categories of NUI Components to 'UI Components' and remove the duplicated UI component guide, such as old 'Button' and old 'Slider'.
2. Change the names :
  1) 'UI Components' -> just 'Components'
  2) 'Natural User Interface'  ->  'NUI'